downstream/modules/eda/proc-eda-set-up-rulebook-activation.adoc

```diff
@@ -26,8 +26,8 @@ Restart policy:: This is a policy to decide when to restart a rulebook.
 ... Never: Never restarts a rulebook when it finishes
 ... On failure: Only restarts when it fails
 Rulebook activation enabled?:: This automatically enables the rulebook activation to run.
-Variables:: Pass extra command line variables to the playbook in the Variables section. These are the `-e` or `--extra-vars` command line parameters for ansible-rulebook. 
-Provide key/value pairs using either YAML or JSON. Refer to the documentation for example syntax.
+Variables:: 
```

Comment:
The proposed text for this field is pending for validation here: https://issues.redhat.com/browse/AAP-14772

Reply:
Waiting for Joe/Ben to confirm the proposed text.

Reply:
Ben confirmed the help text should be: 
"The variables for the rulebook in a json/yaml format. The content would be equivalent to the file passed through the '--vars' flag of ansible-rulebook command"